packages/api/sequelize.config.cjs

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  development: {
+    username: process.env.POSTGRES_USER,
+    password: process.env.POSTGRES_PW,
+    database: process.env.POSTGRES_DB,
+    host: process.env.POSTGRES_HOST,
```

Comment:
Can we also add `port: process.env.POSTGRES_PORT` to this. I can't imagine we would set it to anything other than the default port `5432` but we do set it in the environment so we ought to use it here as well.